WordPress/wordpress-playground
# Throw an error when activating a theme or plugin that doesn't exist

Fixes #1347

## What is this PR doing?
 
Throw an error when activating a theme or plugin that doesn't exist.

## What problem is it solving?

Users currently don't know when the activate plugin and theme steps fail. 

## How is the problem addressed?

By checking if the file exists before trying to activate it.

## Testing Instructions

- Checkout this branch
- [Run this blueprint t](http://localhost:5400/website-server/#{%20%22landingPage%22:%20%22/wp-admin/site-editor.php%22,%20%22preferredVersions%22:%20{%20%22php%22:%20%227.4%22,%20%22wp%22:%20%226.5%22%20},%20%22phpExtensionBundles%22:%20[%20%22kitchen-sink%22%20],%20%22features%22:%20{%20%22networking%22:%20true%20},%20%22steps%22:%20[%20{%20%22step%22:%20%22login%22,%20%22username%22:%20%22admin%22,%20%22password%22:%20%22password%22%20},%20{%20%22step%22:%20%22activatePlugin%22,%20%22pluginName%22:%20%22Gutenberg%22,%20%22pluginPath%22:%20%22/wordpress/wp-content/plugins/gutenberg%22%20}%20]%20})o activate an unexisting plugin
- You should see the error modal with an error about the failed plugin install step
- [Run this blueprint t](http://localhost:5400/website-server/#{%20%22steps%22:%20[%20{%20%22step%22:%20%22activateTheme%22,%20%22themeFolderName%22:%20%22storefront%22%20}%20]%20})o activate an unexisting theme
- You should see the error modal with an error about the failed theme install step
